ai-catalog/app/src/main/java/com/android/ai/catalog/domain/SampleCatalog.kt

```diff
@@ -39,16 +39,16 @@ import com.android.ai.theme.extendedColorScheme
 @RequiresPermission(Manifest.permission.RECORD_AUDIO)
 val sampleCatalog = listOf(
     SampleCatalogItem(
-        title = R.string.gemini_image_chat,
```

Comment:
Why all these renames? I didn't see any info about them in your PR description. Happy to apply the changes if we think they improve the code base, but not sure if that's the case.